sdk/eventhub/azure-eventhub/stress/scripts/logger.py

```diff
@@ -3,43 +3,65 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+import os
 import sys
 import logging
-from logging.handlers import RotatingFileHandler
+from logging.handlers import TimedRotatingFileHandler
 
 from opencensus.ext.azure.log_exporter import AzureLogHandler
 
 
-def get_base_logger(log_filename, logger_name, level=logging.ERROR, print_console=False, log_format=None,
-                    log_file_max_bytes=20 * 1024 * 1024, log_file_backup_count=3):
+
+def get_base_logger(log_filename, logger_name, level=logging.ERROR, print_console=False, log_format=None):
     logger = logging.getLogger(logger_name)
     logger.setLevel(level)
-    formatter = log_format or logging.Formatter('%(asctime)s %(name)-12s %(levelname)-8s %(message)s')
+    formatter = log_format or logging.Formatter('%(asctime)s - [%(thread)d.%(threadName)s] - %(name)s - %(levelname)s - %(message)s')
 
     if print_console:
         console_handler = logging.StreamHandler(stream=sys.stdout)
         if not logger.handlers:
             console_handler.setFormatter(formatter)
             logger.addHandler(console_handler)
+    else:
+        # rotated hourly if small file, o/w rotated bi-hourly
```

Comment:
rotates as in prints to multiple files for the one producer?

Reply:
yep! and separate files for consumer